UI/src/assetTypeConfiguration.js

```diff
@@ -849,10 +858,12 @@
         ],
         formLabels: {
           title: 'Rajoitus',
-          showUnit: true
+          showUnit: true,
+          manyFloatingAssetsLabel: 'TR suurin sallittu leveys'
```

Comment:
we should probably have a singleFloatingAssetsLabel as well. 
manyFloatingAssetsLabel: 'TR suurin sallitut leveydet'
singleFloatingAssetLabel: 'TR suurin sallittu leveys'